composer.json

```diff
@@ -80,7 +80,8 @@
     },
     "conflict": {
         "doctrine/dbal": "<3.4",
-        "doctrine/orm": "<2.14"
+        "doctrine/orm": "<2.14",
+        "symfony/dom-crawler": ">7.4"
```

Comment:
it can be removed after this issue be solved

https://github.com/symfony/panther/issues/685